tests/rgw/sanity_rgw.py

```diff
@@ -86,7 +86,11 @@ def run(ceph_cluster, **kw):
     test_config = {"config": config.get("test-config", {})}
     rgw_node = rgw_ceph_object.node
     client_node = client_ceph_object.node
-    run_on_rgw = config.get("run-on-rgw", False)
+    run_on_rgw = (
+        True
+        if ceph_cluster.rhcs_version.version[0] == 4
```

Comment:
would it be good to use `ceph_cluster.rhcs_version.version.startswith('4')` instead of indexing?